nx.json

```diff
@@ -41,7 +41,7 @@
         "accessToken": "OTg2OGFkNmMtNzA5Zi00MjBiLWFhMmQtOGYwNTQ1MjM1ZjQ3fHJlYWQtd3JpdGU=",
         "canTrackAnalytics": false,
         "showUsageWarnings": true,
-        "parallel": false
+        "parallel": 1
```

Comment:
question: Do we specifically need serial task running everywhere, including locally?

suggestion: Remove this setting to use the default parallelization (3 tasks in parallel)
```suggestion
```
However, we might want or need to change our CI jobs to do this.

Reply:
agree! I guess you already made sure the CI is precisely specifying everything, right?

Reply:
:+1: but let's do this in another PR

Reply:
additional issue created https://github.com/rx-angular/rx-angular/issues/1527